rotkehlchen/tasks/assets.py

```diff
@@ -381,3 +391,57 @@ def update_aave_v3_underlying_assets(chains_aggregator: 'ChainsAggregator') -> N
             name=DBCacheStatic.LAST_AAVE_V3_ASSETS_UPDATE,
             value=ts_now(),
         )
+
+
+def maybe_detect_new_tokens(database: 'DBHandler') -> None:
```

Comment:
please add a docstring explaining the what this function does and the algorithm it follows (short explanation)